docs/content/specs/shared/_index.md

```diff
@@ -118,7 +118,7 @@ To highlight that AVM modules use telemetry, an information notice **MUST** be i
 {{< expand "➕ Telemetry Information Notice" "expand/collapse" >}}
 
 {{< hint type=note >}}
-The following information notice is automatically at the bottom of the `README.md` file of the module when using the [`avm/utilities/tools/Set-AVMModule.ps1`](https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/tools/Set-AVMModule.ps1) tool
+The following information notice with header 'Data Collection' is automatically at the bottom of the `README.md` file of the module when using the [`avm/utilities/tools/Set-AVMModule.ps1`](https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/tools/Set-AVMModule.ps1) tool.
```

Comment:
Please can we add the Terraform equivalent note for Terraform:

The `_footer.md` file in the Terraform template repo contains the required wording and this will be added to the README.md when using the `make docs` command.

Reply:
@matt-FFFFFF please check if the following works for you: 
![image](https://github.com/Azure/Azure-Verified-Modules/assets/5365358/529918dd-f769-461c-a1cf-bb1b9599813c)

Reply:
Perfect thanks!